quickwit/quickwit-indexing/src/actors/indexing_service.rs

```diff
@@ -1200,7 +1221,9 @@ mod tests {
 
     #[tokio::test]
     async fn test_indexing_service_apply_plan() {
-        const PARAMS_FINGERPRINT: u64 = 3865067856550546352;
+        const PARAMS_FINGERPRINT_INGEST_API: u64 = 1637744865450232394;
+        const PARAMS_FINGERPRINT_SOURCE_1: u64 = 1705211905504908791;
+        const PARAMS_FINGERPRINT_SOURCE_2: u64 = 8706667372658059428;
```

Comment:
the change of fingerprints will force all pipelines to restart on existing clusters

Reply:
updated the migration guide to recommend scaling down indexers